index.js

```diff
@@ -1 +1,82 @@
-console.log('hello world');
+import { read, add, edit } from './jsonFileStorage.js';
+
+const command = process.argv[2];
+
+const showList = () => {
+  // read(filename, callback)
+  // callback(null, jsonContentObj);
+
+  read('data.json', (readErr, jsonContentObj) => {
+    if (readErr) {
+      console.error('Read error', readErr);
+      return;
+    }
+
+    // define to do list items
+    const toDoList = jsonContentObj['To Do'];
+    console.log('To-Do-List:');
+    // console log to do list items
+    for (let i = 0; i < toDoList.length; i += 1) {
+      console.log(`${i + 1}. ${toDoList[i]}`);
+    }
```

Comment:
you can consider using a `forEach` method here instead

```suggestion
   toDoList.forEach((item, i) => console.log(`${i+ 1}. ${item}`)
```